providers/amazon/images/ivy-kubernetes/provision.yml

```diff
@@ -7,7 +7,8 @@
     - system-base
     - docker
     - kubernetes
+    - nvidia-docker
```

Comment:
We recently removed the `ivy-kubernetes` image, and moved to versioned roles. You should add this line to the `ivy-kubernetes-1.21` image now. Sorry for the confusion that I caused with my prior comment.